suzieq/engines/pandas/network.py

```diff
@@ -179,7 +206,16 @@ def _find_addr_arp(self, addr: str, **kwargs) -> pd.DataFrame:
                      ifname=[oif])
 
             if not ifdf.empty:
-                if 'vlan' not in ifdf.type.unique():
+                netaddr = ip_address(row.ipAddress)
+                # The code below checks if the provided address belongs to
+                # the interface subnet. If it does, its a bridged address
+                # else its a routed address i.e. you use routing to reach it.
+                if not ifdf.ipAddressList.apply(
+                        lambda subnets, netaddr:
+                        False if not subnets.any() else
+                        any(addr in ip_network(subnet, strict=False)
```

Comment:
Both `ip_address(row.ipAddress)` and `ip_network(subnet, strict=False)` expect a `_RawIpAddress`. `ip_address` seems to work also with a string, but `ip_network` raises this exception:
```shell
luca> network find address='10.0.0.11'                                                                                                                                                                                                                                                                Error: 1
                                             error
0  ERROR: 'str' object has no attribute '_version'
```

Reply:
Fixed. Typo when I changed var names.